frontend/src/scenes/settings/project/ProjectMove.tsx

```diff
@@ -0,0 +1,126 @@
+            onClose={!projectBeingMovedLoading ? () => setIsOpen(false) : undefined}
+            closable={!projectBeingMovedLoading}
+            maxWidth="30rem"
+            footer={
+                <>
+                    <LemonButton
+                        disabledReason={projectBeingMovedLoading && 'Moving...'}
+                        type="secondary"
+                        onClick={() => setIsOpen(false)}
+                    >
+                        Cancel
+                    </LemonButton>
+                    <LemonButton
+                        type="secondary"
+                        disabled={!isConfirmed}
+                        loading={projectBeingMovedLoading}
+                        data-attr="move-project-ok"
+                        status="danger"
+                        onClick={currentProject ? () => moveProject(currentProject, organization.id) : undefined}
+                    >{`Move ${currentProject ? currentProject.name : 'the current project'}`}</LemonButton>
+                </>
+            }
+            isOpen={isOpen}
+        >
+            <p>
+                Moving a project will mean all original organization members will lose access including via things like
+                API keys unless they also are part of the new organization.
+            </p>
+            <p>
+                Please type <strong>{currentProject ? currentProject.name : "this project's name"}</strong> to confirm.
+            </p>
+            <LemonInput
+                type="text"
+                onChange={(value) => {
+                    if (currentProject) {
+                        setConfirmed(value.toLowerCase() === currentProject.name.toLowerCase())
+                    }
+                }}
+            />
+        </LemonModal>
+    )
+}
+
+export function ProjectMove(): JSX.Element {
+    const { currentProject } = useValues(projectLogic)
+    const { otherOrganizations } = useValues(userLogic)
+    const [isModalVisible, setIsModalVisible] = useState(false)
+
+    const restrictedReason = useRestrictedArea({
+        minimumAccessLevel: OrganizationMembershipLevel.Admin,
+        scope: RestrictionScope.Project,
+    })
+
+    const [targetOrganization, setTargetOrganization] = useState<OrganizationBasicType | null>(null)
+
+    return (
+        <>
+            <p>
+                Move <b>{currentProject?.name}</b> to another organization. You must be at least an Admin of both
+                organizations. Once this is done, all access and settings applying to the current Organization will no
+                longer apply.
+            </p>
+
+            <div className="flex items-center gap-2">
+                <LemonSelect
+                    options={otherOrganizations.map((o) => ({
+                        label: o.name,
+                        value: o.id,
+                    }))}
+                    placeholder="Select an organization"
```

Comment:
A bit of extra clarity:
```suggestion
                    placeholder="Select target organization"
```